modules/apps/content-dashboard/content-dashboard-web/src/main/java/com/liferay/content/dashboard/web/internal/display/context/ContentDashboardAdminDisplayContext.java

```diff
@@ -137,7 +142,11 @@ public String getAuthorItemSelectorURL() throws PortalException {
 			return _data;
 		}
 
-		_data = Collections.singletonMap("props", _getProps());
+		_data = HashMapBuilder.<String, Object>put(
+			"context", _getContext()
```

Comment:
We usually use in portal 'textDirection' as the API field to say if the text is "rtl" or "ltr"